### PR TITLE
Sample application to use the implementation of large audit event client 

### PR DESCRIPTION
**Pull request by smahapatra86**
- Added sample application to use the large audit event client

### CODE REVIEW

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                AuditEventTopic = config.Ingestion.AuditEventTopic,
+                EventHubConnectionString = config.Ingestion.ConnectionString,
+                EnrichFromLogContext = config.Client.EnrichFromLogContext,
+                StorageAccountName = config.Ingestion.BlobAccountName,
+                StorageConnectionString = config.Ingestion.BlobConnectionString,
+                StorageContainerName = config.Ingestion.BlobContainerName
+            };
+
+            var events = new List<AuditEvent>();
+            for (int i = 0; i < 100000; i++)
```
Comment (avin101) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs]: We should probably apply smaller number of events?

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                using (Serilog.Context.LogContext.PushProperty("GroupId", groupId))
+                {
+                    Enumerable
+                        .Range(0, config.Ingestion.NumberOfEventsToSend)
+                        .AsParallel()
+                        .WithDegreeOfParallelism(config.Ingestion.NumberOfThreads)
+                        .WithExecutionMode(ParallelExecutionMode.ForceParallelism)
+                        .Select(i =>
+                        {
+                            audit.Write("Pre release testing with number {IterationNum} from {Application} v{Version} data {Data}", i, name, version, events);
```
Comment (avin101) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs]: Maybe we should change the text "Pre release testing" to something more generic?

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/AuditEvent.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+        public Guid Name { get; set; }
+
+        public Guid Department { get; set; }
+
+        public Guid Age { get; set; }
+
+        public AuditEvent()
+        {
+            this.Id = Guid.NewGuid();
```
Comment (manikraja) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/AuditEvent.cs]: why do we initialize it in constructor? can initialization be like the properties declared in sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/ProgramConfig.cs

**Review by manikraja — COMMENTED** [review]
Added a comment to AuditEvent.cs

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                EventSource = config.Ingestion.EventSource ?? $"{typeof(Program).Assembly.GetName().Name} on {Environment.MachineName}",
+                AuditEventTopic = config.Ingestion.AuditEventTopic,
+                EventHubConnectionString = config.Ingestion.ConnectionString,
+                EnrichFromLogContext = config.Client.EnrichFromLogContext,
+                StorageAccountName = config.Ingestion.BlobAccountName,
+                StorageConnectionString = config.Ingestion.BlobConnectionString,
+                StorageContainerName = config.Ingestion.BlobContainerName
+            };
+
+            // Create data for audit event
```
Comment (thecorrado) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs:39]: Those sample don't provide any value in terms of testing a new _client_. As an event data I would use some bigger payload (>256KB) to test both saving blob in storage and sending event.

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+            }
+
+            using (var client = new Kmd.Logic.Audit.Client.SerilogLargeAuditEvents.SerilogLargeAuditEventClient(clientConfig))
+            {
+                var audit = (IAudit)client;
+                var name = typeof(Program).Assembly.GetName().Name;
+                var version = typeof(Program).Assembly.GetName().Version;
+
+                Console.WriteLine(
+                    "Sending {0} ({3} threads) audit events to {1} at {2}",
```
Comment (thecorrado) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs]: Write storage/container name too.

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+                    config.Ingestion.NumberOfEventsToSend,
+                    clientConfig.AuditEventTopic,
+                     clientConfig.EventHubConnectionString,
```
Comment (thecorrado) [inline sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs]: align the line

**Review by thecorrado — CHANGES_REQUESTED** [review]

File: sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs (hunk 4 above)
Comment (smahapatra86) [reply sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs:39]: Increased loop count to increase payload to 2 MB

**Review by thecorrado — APPROVED** [review]
LGTM 👍

**Pull request merged**